src/clients/aggregator_client/api_types.rs

```diff
@@ -173,11 +167,21 @@ pub struct TaskResponse {
     pub time_precision: JanusDuration,
     pub tolerable_clock_skew: JanusDuration,
     pub collector_hpke_config: HpkeConfig,
-    pub aggregator_auth_tokens: Vec<String>,
-    pub collector_auth_tokens: Vec<String>,
+    pub aggregator_auth_token: String,
+    pub collector_auth_token: String,
```

Comment:
I think this needs to be `collector_auth_token: Option<String>`. In the case where you are creating a task in a helper, the response will not contain a collector auth token. Since the collector never talks to the helper, it doesn't make sense for the helper to generate that, nor should the helper ever see the token used by the leader and collector.

Reply:
resolved in a701c200f56c41f791441b324bae08d8e424f712